bootcd/rpms/genesis_scripts/src/root-bash_profile

```diff
@@ -5,5 +5,5 @@ if [[ ! -z $GENESIS_MODE ]] && [ ! -e "/var/run/bootloader-autorun.lock" ]
   then
     touch /var/run/bootloader-autorun.lock
     # disable task execution prompting when doing initial target run
-    GENESIS_PROMPT_TIMEOUT=0  /usr/bin/genesis-bootloader | tee /var/log/genesis-bootloader.log
+    GENESIS_PROMPT_TIMEOUT=0  /usr/bin/genesis-bootloader 2>&1 | tee /var/log/genesis-bootloader.log
```

Comment:
We might want to do
GENESIS_PROMPT_TIMEOUT=0  /usr/bin/genesis-bootloader 2>&1 | tee /var/log/genesis-bootloader.log | tee /dev/console
to force the output onto the console, but that will make working on the console hard/noisy and may not give us the desired output on bare metal vs virtualbox consoles.